python/lsst/meas/extensions/psfex/psfexStarSelector.py

```diff
@@ -70,11 +70,13 @@ class PsfexStarSelectorConfig(BaseSourceSelectorTask.ConfigClass):
         doc="Max number of bad pixels ",
         default=0,
         check=lambda x: x >= 0,
+        deprecated="This field has never worked. Will be removed after v21."
```

Comment:
In the deprecation message state that this config option is actually not used anywhere in the code any more.

Reply:
I mean, if anyone had set it, they'd have gotten an `AssertionError`, but I'll expand the message.